test/test_ops.py

```diff
@@ -676,6 +667,43 @@ def test_boxes_shape(self):
         self._helper_boxes_shape(ops.ps_roi_align)
 
 
+@pytest.mark.parametrize(
```

Comment:
This could be in `RoIOpTester` as a base test, but I personally find it really hard to figure out what is being tested with such inheritance structure. So I'm opting for a self-contained parametrized test here.

(The test inheritance structure predates the adoption of pytest.)